sigma/backends/loki/loki.py

```diff
@@ -297,14 +297,14 @@ def set_expression_templates(negated: bool) -> None:
 
     # Loki-specific functionality
     add_line_filters: bool = False
-    case_insensitive: bool = True
+    case_sensitive: bool = True

```

Comment:
I realise this is being set to the default value correctly in the constructor, but I'd prefer it if this could be updated to default to `false` here too, just to avoid issues down the line.

Reply:
Mostafa got here first :)